app.py

```diff
@@ -38,6 +43,23 @@ def get_stream():
         return streams[0]
     return '{}'
 
+@app.route('/streamtrainId/<trainId>')
+def get_stream_train(trainId):
+    print("Trainid: ", trainId)
```

Comment:
Unnecessary prints makes log analysis hard; if it needs to be logged let's set up a propper logging endpoint like in `scanner.py`.